index.d.ts

```diff
@@ -0,0 +1,144 @@
+type Direction = 'top' | 'top-right' | 'right' | 'bottom-right' | 'bottom' | 'bottom-left' | 'left' | 'top-left'
+
+export class js {
+
+  /**
+  * Sets the collision grid that EasyStar uses.
```

Comment:
Yes I see it. Lets remove the `.ts` from this PR, and it will be added once we merge the other one. This way we can keep this PR focused around the directions.